src/main/kotlin/no/nav/familie/ef/sak/iverksett/SimuleringService.kt

```diff
@@ -54,12 +55,11 @@ class SimuleringService(private val iverksettClient: IverksettClient,
                                                        eksternFagsakId = fagsak.eksternId.id
 
                 )
-        val forrigeTilkjentYtelse = tilkjentYtelseService.finnSisteTilkjentYtelse(fagsakId = behandling.fagsakId)?.tilIverksett()
+        val forrigeBehandlingId = behandlingService.hentForrigeBehandlingId(behandling.fagsakId)
 
         return iverksettClient.simuler(SimuleringDto(
                 nyTilkjentYtelseMedMetaData = tilkjentYtelseMedMedtadata,
-                forrigeTilkjentYtelse = forrigeTilkjentYtelse
-
+                forrigeBehandlingId = forrigeBehandlingId
         ))
     }
```

Comment:
Lurer på hvorfor sjekker vi for behandlingType.blankett to ganger dvs linje 45 og linje 39. Er det ikke nok bare å sjekke det på linje 39?

Reply:
Bra catch, tenker også at den på rad 45 burde kunne fjernes 👍